src/controllers/SitemapController.php

```diff
@@ -61,27 +63,93 @@ private function buildSitemapfile($sitemapFile)
                 'is_offline' => false,
                 'is_draft' => false,
             ])->with(['navItems', 'navItems.lang']);
-            
+
             if (!$this->module->withHidden) {
                 $query->andWhere(['is_hidden' => false]);
             }
-            
+
             foreach ($query->each() as $nav) {
                 /** @var Nav $nav */
 
+                if ((Config::findOne([
+                    'name' => 'httpExceptionNavId',
```

Comment:
You could use this constant in favor of the variable string:
https://github.com/luyadev/luya-module-cms/blob/248c1803b246a40bbae482ff6b409720dc469211/src/models/Config.php#L20

(like on line#145)

Reply:
fixed by 3e95b6ad64ae85271faf6e40c0e348bbda8dcfc5